src/components/navbar/Navbar.tsx

```diff
@@ -32,19 +33,28 @@ const Navbar: React.FC = () => {
           {isOpen ? <X className="size-6" /> : <Menu className="size-6" />}
         </button>
       </div>
+
       <div
         data-testid="menu"
         id="navbar-menu"
         className={`flex-col md:flex-row mt-6 md:mt-0 items-center gap-y-3 md:gap-2 ${
           isOpen ? "flex" : "hidden"
         } md:flex`}
       >
-        <Button variant="secondary">
-          <Link to="/registration">Inscription</Link>
-        </Button>
-        <Button variant="outline">
-          <Link to="/login">Connexion</Link>
-        </Button>
+        {isUserConnected ? (
+          <Button>
+            <Link to="/">Déconnexion</Link>
+          </Button>
+        ) : (
+          <>
+            <Button variant="secondary">
+              <Link to="/registration">Inscription</Link>
+            </Button>
+            <Button variant="outline">
+              <Link to="/login">Connexion</Link>
```

Comment:
La route pour la page de Connexion c'est:
`<Link to="/connexion">Connexion</Link>`
Attention de changer dans App.tsx egalement